lib/workers/repository/process/lookup/index.ts

```diff
@@ -154,6 +154,15 @@ export async function lookupUpdates(
         res.updates.push(rollback);
       }
       let rangeStrategy = getRangeStrategy(config);
+
+      if (config.replacementName && !config.replacementVersion) {
+        res.updates.push({
+          updateType: 'replacement',
+          newName: config.replacementName,
+          newValue: currentValue!,
```

Comment:
what happens when we've a docker image without tag and digest. `currentValue` can probably be undefined in that case.

Reply:
Just did a quick test and that scenario at the moment isn't handled 😅

Reply:
updated to handle this scenario.  testing on two real repositories also